lib/smartsheet/http_client.ex

```diff
@@ -110,10 +110,9 @@ defmodule Smartsheet.HttpClient do
     |> URI.to_string()
   end
 
-  @api_key Application.get_env(:smartsheet, :api_key) || System.get_env("SMARTSHEET_API_KEY")
```

Comment:
i had a feeling this change would be problematic but i went with it anyway lol. why do i never listen to my gutttt